examples/09_Kotlin_JS/06_HtmlBuilder.md

```diff
@@ -99,11 +131,11 @@ class LI() : BodyTag("li")
 class P() : BodyTag("p")
 class H1() : BodyTag("h1")
```

Comment:
Add these classes for proper parsing of \<ul> and \<li>:

```suggestion
class H1() : BodyTag("h1")
class LI() : BodyTag("li")

class UL : BodyTag("ul") {
    fun li(init: LI.() -> Unit) = initTag(LI(), init)
}

```

Reply:
This snippet is already present in my source